std/math/emulated/element.go

```diff
@@ -33,26 +33,17 @@ type Element[T FieldParams] struct {
 	internal bool
 }
 
-// NewElement builds a new emulated element from input. The inputs can be:
-//   - of type Element[T] (or a pointer to it). Then, the limbs are cloned and packed into new Element[T],
-//   - integer-like. Then it is cast to [*big.Int], decomposed into limbs and packed into new Element[T],
-func NewElement[T FieldParams](v interface{}) Element[T] {
-	if e, ok := v.(Element[T]); ok {
-		return *e.copy()
-	} else if e, ok := v.(*Element[T]); ok {
-		return *e.copy()
-	} else if frontend.IsCanonical(v) {
-		// We can not force that v is correct width. Better use PackFullLimbs
-		panic("can not enforec limb width. Use PackLimbs instead")
-	} else if v == nil {
+// ValueOf returns an Element[T] from a constant value.
+// The input is converted to *big.Int and decomposed into limbs and packed into new Element[T].
+func ValueOf[T FieldParams](constant interface{}) Element[T] {
+	if constant == nil {
 		r := newConstElement[T](0)
-		r.internal = false
-		return *r
-	} else {
-		r := newConstElement[T](v)
-		r.internal = false
+		// r.internal = false
```

Comment:
Is the line intentionally commented? I think we still need for enforcing the widths of the limbs on the first use of Element?

Reply:
why though; this path just init constants. That goes through decompose big int method. We are already guaranteed the width of the element is correct ,since it's constant and built by the package. 
Now if someone does 
```
myConstant := emulated.NewElment[T](42)
myConstant.Limbs[0] = someVariable
```
then ...